includes/CategoryPage.php

```diff
@@ -60,6 +60,11 @@ function view() {
 		parent::view();
 
 		if ( NS_CATEGORY == $title->getNamespace() ) {
+			/* Wikia change - begin */
```

Comment:
I'm not sure if this is a good example, but Wall uses `onArticleViewHeader` to set the status code. I'd rather not modify the MW files if not needed.

Long term idea: merge CategoryTree, CategoryPagination, CategoryGalleries and CategoryExhibitions into one WikiaCategory extension and use it to hold the hook code.